src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -58,10 +58,24 @@
       DestinationFiles="$(OutputPath)..\..\..\mandroid\MULTIDEX_JAR_LICENSE" />
   </Target>
 
-  <Target Name="_BuildMonoSymbolicate">
-    <MSBuild
-        Projects="..\..\external\mono\mcs\tools\mono-symbolicate\monosymbolicate.csproj"
-        Properties="OutputPath=..\..\..\..\..\bin\$(Configuration)\lib\mandroid\;Configuration=$(Configuration);AssemblyName=mono-symbolicate"
+  <Target Name="_BuildMonoSymbolicate"
+      Inputs="..\..\external\mono\mcs\tools\mono-symbolicate\mono-symbolicate.exe.sources"
+      Outputs="$(OutputPath)..\..\..\..\lib\mandroid\mono-symbolicate.exe"
+      >
+    <ReadLinesFromFile  
+            File="..\..\external\mono\mcs\tools\mono-symbolicate\mono-symbolicate.exe.sources" >  
+        <Output  
+            TaskParameter="Lines"  
+            ItemName="SourceFiles"
+        />  
+    </ReadLinesFromFile>
+    <Csc
+        Sources="@(SourceFiles->'..\..\external\mono\mcs\tools\mono-symbolicate\%(Identity)')"
```

Comment:
This, and other locations, should use `$(MonoSourceFullPath)`, not hardcode into `..\..\external\...`.